docs/conf.py

```diff
@@ -13,11 +13,11 @@
 html_static_path = ["_static"]
 html_logo = "_static/logo_wide.svg"
 html_favicon = "_static/logo_square.svg"
+html_css_files = [
```

Comment:
The sphinx-book-theme and pydata-sphinx-theme both already ship with fontawesome, so this does not need to be added for them

Reply:
According to pydata theme docs, FA v5 is included, but I believe this statement is outdated. That theme's [webpack config](https://github.com/pydata/pydata-sphinx-theme/blob/761bec818006a51352e20a1a8fb2ab5d2f9ec62e/webpack.config.js#L68) seems setup to use the latest FA release.

The sphinx-book-theme seems to depend on the pydata theme's templates, so I think the latest FA release is used in the sphinx-book-theme also.